gradle-plugins/compose/src/test/test-projects/misc/jvmOnlyResources/expected/Res.kt

```diff
@@ -23,6 +24,16 @@ internal object Res {
      */
     public suspend fun readBytes(path: String): ByteArray = readResourceBytes("" + path)
 
+    /**
+     * Returns the URI string of the resource file at the specified path.
+     *
+     * Example: `val uri = Res.getUri("files/key.bin")`
+     *
+     * @param path The path of the file in the compose resource's directory.
+     * @return The URI string of the file.
+     */
+    public fun getUri(path: String): String = getResourceUri("" + path)
```

Comment:
Should not we add test that calls getUri (we may check that resulting URI contains original path)?